pkg/nsx/services/common/store.go

```diff
@@ -152,3 +152,42 @@ func (service *Service) InitializeResourceStore(wg *sync.WaitGroup, fatalErrors
 		}
 	}
 }
+
+// InitializeVPCResourceStore is the method to query all the various VPC resources from nsx-t side and
+// save them to the store, we could use it to cache all the resources when process starts.
+func (service *Service) InitializeVPCResourceStore(wg *sync.WaitGroup, fatalErrors chan error, org string, project string, resourceTypeValue string, store Store) {
```

Comment:
could InitializeVPCResourceStore merged with InitializeResourceStore? if org/project == nil, call the service.NSXClient.QueryClient.List, else call the service.NSXClient.VPCQueryClient.List

Reply:
I absorb a common method in the latest version, take a look